lib/internal/test_runner/runner.js

```diff
@@ -424,9 +424,12 @@ function watchFiles(testFiles, opts) {
 }
 
 function run(options) {
-  if (options === null || typeof options !== 'object') {
-    options = kEmptyObject;
+  if (options != null) {
+    validateObject(options, 'options');
   }
+
+  options = options ?? kEmptyObject;
```

Comment:
If we're going to make this a breaking change anyway, could we make `kEmptyObject` a default argument and drop  the `if` statement and this line.

Another option is to make this line an `else` so we don't need to execute it all the time.

Reply:
Note that would have different behavior for null (which i suppose is why it’s breaking)

Reply:
done